tests/shared/src/test/scala/scala/meta/tests/parsers/StarParameterSuite.scala

```diff
@@ -0,0 +1,136 @@
+package scala.meta.tests
+package parsers
+
+import scala.meta._, Defn.Class
+import scala.meta.dialects.Scala211
+
+class StarParameterSuite extends ParseSuite {
+
+  test("star parameter single argument") {
```

Comment:
`vararg parameter single argument` maybe? And with all the other tests.

Reply:
right, replaced 'star' with 'vararg', also renamed a file(that's why this comment might get lost sorry)